content/applications/general/users/facebook.rst

```diff
@@ -0,0 +1,121 @@
+   Meta account. If this email address is not regularly monitored, it may be wise to use another
+   email address.
+
+Click :guilabel:`Next`. Review the :guilabel:`Publishing requirements`, the :guilabel:`Meta
+Platform Terms` and :guilabel:`Developer Policies`. Then, click :guilabel:`Create app`.
+
+.. important::
+   Clicking :guilabel:`Create app` may require password re-entry.
+
+Customize app
+-------------
+
+After the new app is created, the :guilabel:`Dashboard` page appears, with a list of steps to be
+completed before the app can be published. From here, click :guilabel:`Customize adding a Facebook
+Login button`.
+
+.. image:: facebook/app-requirements.png
+   :align: center
+   :alt: The App Dashboard in the Meta for developers platform.
+
+On the :guilabel:`Customize` page, click :guilabel:`Settings`.
+
+In the :guilabel:`Valid OAuth Redirect URIs` field, enter `https://<odoo base
+url>/auth_oauth/signin`, replacing `<odoo base url>` with the URL of the applicable database. Click
+:guilabel:`Save changes` when finished.
+
+Configure settings
+------------------
+
+At the far left of the page, click :menuselection:`App settings --> Basic`. This page contains
+additional settings that are required before the app can be submitted for approval.
+
+In the :guilabel:`Privacy Policy URL` field, enter `https://www.odoo.com/privacy`.
+
+.. note::
+   `<https://www.odoo.com/privacy>`_ is the default privacy policy for databases hosted on Odoo.com.
+
+Click the :guilabel:`App Icon` field to open a file upload window. From here, select and upload an
+app icon.
+
+In the :guilabel:`User data deletion` field, enter
+`https://www.odoo.com/documentation/17.0/administration/odoo_accounts.html`.
+
+.. note::
+   This document provides instructions on how a user can delete their Odoo account.
+
+Lastly, click the :guilabel:`Category` field and select :guilabel:`Business and pages` from the
+drop-down menu.
+
+Click :guilabel:`Save changes`.
+
+.. image:: facebook/app-id.png
+   :align: center
+   :alt: An exampled of the Basic Settings page in the Meta for developers platform.
+
+.. _users/app-id:
+
+Capture app ID
+--------------
+
+After the app is created, and approved, select and copy the :guilabel:`App ID`. Paste this
+information on a clipboard or notepad file, as it is needed in a later step to complete the setup.
+
+Publish
+-------
+
+On the left side of the page, click :guilabel:`Publish`. Depending on the status of the connected
+*Facebook* account, additional verification and testing steps may be required, and will be listed
+on this page.
+
+After reviewing the information, click :guilabel:`Publish`.
+
+.. seealso::
+   Additional information regarding Meta App Development, including further details on building,
+   testing, and use cases, can be found in the `Meta for developers documentation
+   <https://developers.facebook.com/docs/development>`_.
```

Comment:
Genuine question: have we decided for the doc style guide that we'll include additional text in seealso admonition blocks? Is it no longer required to *only* have link text in these?

Only link text:
![image](https://github.com/user-attachments/assets/79672de8-4471-42ee-8dac-be40c4d93b02)

Other text, including link:
![image](https://github.com/user-attachments/assets/da562f1f-e15e-493e-a141-c1a2578405ca)

Reply:
I confirmed with Sam that this is ok for seealso blocks in general to include additional content beyond just the links. And especially in this case, I wanted to add a bit more context so they understand that there is only so much information about the Meta side of things we can provide, if they have more technical questions in the Meta dashboard, they really need to go to their resources :+1: